src/app.ts

```diff
@@ -33,7 +33,8 @@ async function main() {
       const manager = getManager();
       // find the user who has a token in their sessions field
```

Comment:
a little confused how this works - the server is created and then a single user's session token is checked?

Reply:
no, server is created with a defined function `currentUserChecker`. every time the `@CurrentUser` decorator is used, the function is called